draft-ietf-sedate-datetime-extended.md

```diff
@@ -117,97 +117,91 @@ informative:
 
 
 This document defines an extension to the timestamp format defined in
-RFC3339 for representing additional information including a time
-zone.
+RFC3339 for representing additional information including a time zone.
 
-It updates RFC3339 in the specific interpretation of the local offset
+It updates {{RFC3339}} in the specific interpretation of the local offset
```

Comment:
FYI, we can't have reference in abstracts.
https://www.rfc-editor.org/rfc/rfc7322#section-4.3